route/build/src/historyFile.f90

```diff
@@ -108,7 +108,7 @@ SUBROUTINE set_compdof_rch(this, compdof_rch, nRch_in)
       integer(i4b),             intent(in)     :: nRch_in          ! total number of reaches
 
       ! initialze iodescRchFlux
-      call pio_decomp(this%pioSystem,    & ! input: pio system descriptor
+      call pio_decomp(this%pioSys,       & ! input: pio system descriptor
                       ncd_float,         & ! input: data type (pio_int, pio_real, pio_double, pio_char)
```

Comment:
This and the next two are the parts where it would be good to be able to select if double or single precision is desired.

Reply:
history file is now using single precision so this should be ncd_float